src/main/java/seedu/address/model/feedtime/FeedTime.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.model.feedtime;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a FeedTime in the address book.
+ * Guarantees: immutable; name is valid as declared in {@link #isValidFeedTime(String)}
+ */
+public class FeedTime {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Feed times should be in the format HHMM, and it should not be blank";
```

Comment:
nitpick: format should be HHmm (lowercase m for minutes since M is for months) 
https://docs.oracle.com/javase/10/docs/api/java/text/SimpleDateFormat.html